README.md

```diff
@@ -210,7 +208,7 @@ test runner.
 You can run all tests with:
 
 ```bash
-$ npm run test
+$ yarn run test
```

Comment:
Is `run` needed here? If yes, shouldn't we add it [here](https://github.com/skroutz/analytics.js/pull/31/commits/aed0fcefc2883150cf7ccf8bbf3479c2b4382f54#diff-04c6e90faac2675aa89e2176d2eec7d8R193), too?